uber/custom_tags.py

```diff
@@ -133,6 +145,7 @@ def email_only(email):
 
 @tag
 class maybe_anchor(template.Node):
+
```

Comment:
just realized you probably meant just the newlines after class declarations and not end function. will remedy immediately.